src/style.css

```diff
@@ -0,0 +1,97 @@
+/* stylelint-disable font-family-no-missing-generic-family-keyword */
+@font-face {
+  font-family: Avenir;
+  src: url("./fonts/Avenir-Book.ttf");
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url("./fonts/Avenir-Heavy.ttf");
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url("./fonts/Avenir.ttc");
+}
+  
+* {
+  box-sizing: border-box;
+}
+
+body {
+  padding-top: 12px;
```

Comment:
better to reset body margins and set padding for form

Reply:
+